src/components/general/BlurCard/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState, type FC } from 'react'
+import { InvertColourButton } from '../Button/styles'
+import { Content } from '../Content'
+import { BlurCard, BlurCardsContainer } from './styles'
+
+export const BlurCards: FC = () => {
+  const [isHovered, setIsHovered] = useState(99)
```

Comment:
99 is a weird default number. A better default number is `-1` or even doing null, we can also add null to the types as well.
you can do:
```
const [ isHovered, setIsHovered] = useState<number | null>(null);
```